lxd/openfga/objects.go

```diff
@@ -0,0 +1,45 @@
+package openfga
+
+import (
+	"fmt"
+)
+
```

Comment:
You've defined new types for Relation and ObjectType, but not for these values, what do they represent and do they warrant their own type too?